crates/sui-node/src/admin.rs

```diff
@@ -87,8 +92,33 @@ pub async fn run_admin_server(node: Arc<SuiNode>, port: u16, filter_handle: Filt
         .unwrap()
 }
 
+#[derive(Deserialize)]
+struct EnableTracing {
+    filter: String,
+    duration: String,
+}
```

Comment:
Is there a way to disable it too?

Reply:
It disables automatically after the duration expires. I guess it wouldn't hurt to be able to disable it actively in case you set too long a duration by accident.

Reply:
added a disable route